app/komponenter/språkvelger/språkvelger.tsx

```diff
@@ -0,0 +1,38 @@
+import { Select } from '@navikt/ds-react';
+import { useSpråk } from '~/root';
+import { LocaleType } from '~/typer/sanity/sanity';
+import css from './språkvelger.module.css';
+import { GlobeIcon } from '@navikt/aksel-icons';
+
+export const Språkvelger = () => {
+  const [, settSpråk] = useSpråk();
+
+  function handleChange(evt: React.ChangeEvent<HTMLSelectElement>) {
+    settSpråk(evt.target.value as LocaleType);
+  }
+
+  return (
+    <>
+      <Select
+        label={<Label />}
+        className={`${css.språkvelger}`}
+        onChange={evt => {
```

Comment:
Her ville jeg vurdert å skrive ut hele parameternavnet i stedet for å bruke en forkortelse. Det er egentlig bare en personlig preferanse, men jeg synes det er en fin vane å ha at man skal tenke at en person som leser koden om 10 år enklest mulig skal forstå hva som står her, og hele variabelnavn gjør koden mer lesbar 😄

Reply:
Høres smart ut! Endret det her [0315140](https://github.com/bekk/nav-familie-endringsmelding/pull/19/commits/031514080fa0a79e77e72960e32cd9efdb053272)